community.hachyderm.io/content/en/docs/blocklists/_index.md

```diff
@@ -0,0 +1,165 @@
+- Balancing open participation with curating a safe space
+
+### Concerns that go into transparency around the Blocklist
+
+Some servers have reasons attached to their moderation action and
+others do not. In addition, we may or may not announce when we
+limit/silence or block/suspend individual instances. Why is this?
+
+When we choose what level of notification to send, and how
+transparent to be, with moderation actions we consider:
+
+1. The impact of the change
+1. The interest level in the change
+1. The risk of publicizing/being transparent about the change
+
+For example, if we were to take moderation action against
+any of the large, popular Mastodon instances we would err on
+the side of transparency as this would be a significant and
+user impacting event.
+
+Whenever we take action on a server that has malicious activity,
+and this can be in the form of attacks on our server or in the
+form of social attacks like stalking and harrassment, we err on
+the side of safety. This means what level of information
+we provide and how loudly (notifications, etc.) we provide
+it will be based on what is safest for all of our users.
+
+The vast, vast majority of instances fall between these two
+extremes and thus the resulting decisions do not fall
+perfectly in the "fully transparent" or "completely silent" buckets.
+This means:
+
+- Servers may not have reasons attached to their moderation
+  decisions, but that doesn't mean they are a security or safety concern.
+  <br />To do this would immediately out them as a security
+  concern, by the absence of information.
+- Not all changes to individual server status will be loudly announced, but some
+  will.<br />
+  We will use the decision making process outlined above when
+  announcing.
+- Not announcing that a server has changed status does not
+  mean that server is or was a cause of concern. It can
+  also, and frequently will, mean that the server size doesn't impact
+  enough of our users to make a large announcement.
+
+## Requesting Moderation Changes for a Server
+
+
+### What to do there is a domain on the Blocklist in error
+
+We are all human and are prone to mistakes. If there is a domain that is
+moderated on our Blocklist that seems to be in error, please open a
+[GitHub Issue in our Community repo](https://github.com/hachyderm/community/issues)
+to request that we take another look at the domain. Please include
+as much relevant context as you can to help us make our decision.
+Note that depending on the circumstances, and as outlined above, we may
+not be able to be fully transparent with our decision - but we commit
+to erring on the side of transparency with these reports as often as possible.
+
+### What to do if you would like us to moderate a server
+
+If there is a server that is not currently moderated, i.e. either
+limited/silenced or banned/suspended, then please _also_ open a
+[GitHub Issue in our Community repo](https://github.com/hachyderm/community/issues)
```

Comment:
this may be higher friction than "filing a report on a user and including references at the instance level".  should we keep some user-report path to reporting open?